package.json

```diff
@@ -43,10 +43,10 @@
     "stylelint-formatter-pretty": "^3.2.0"
   },
   "dependencies": {
-    "@patternfly/patternfly": "5.0.0-alpha.64",
-    "@patternfly/react-core": "5.0.0-alpha.115",
-    "@patternfly/react-icons": "5.0.0-alpha.19",
-    "@patternfly/react-styles": "5.0.0-alpha.16",
+    "@patternfly/patternfly": "5.0.0-prerelease.8",
```

Comment:
Okay, let's use this opportunity to see how/where npm-update screws up. This is most definitively not the latest version. See e.g. also https://github.com/cockpit-project/cockpit/pull/19181